tests/unit_tests/test_hybrid_search_return_fields.py

```
@@ -0,0 +1,208 @@
+import sys
+from pathlib import Path
+
+import pytest
+
+# Ensure local src/ is on sys.path so we import the in-repo pyseekdb,
+# not an already-installed version in the virtualenv.
+project_root = Path(__file__).parent.parent.parent
+src_root = project_root / "src"
+sys.path.insert(0, str(src_root))
+
+from pyseekdb.client.client_base import BaseClient  # noqa: E402
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)